.mergify.yml

```diff
@@ -264,22 +264,27 @@ pull_request_rules:
       - "check-success=crds-gen"
       - "check-success=docs-check"
       - "check-success=pylint"
-      - "check-success=canary"
-      - "check-success=raw-disk"
-      - "check-success=two-osds-in-device"
-      - "check-success=osd-with-metadata-device"
-      - "check-success=encryption"
-      - "check-success=lvm"
-      - "check-success=pvc"
-      - "check-success=pvc-db"
-      - "check-success=pvc-db-wal"
-      - "check-success=encryption-pvc"
-      - "check-success=encryption-pvc-db"
-      - "check-success=encryption-pvc-db-wal"
-      - "check-success=encryption-pvc-kms-vault-token-auth"
-      - "check-success=encryption-pvc-kms-vault-k8s-auth"
-      - "check-success=lvm-pvc"
-      - "check-success=rgw-multisite-testing"
+      - "check-success=canary (quay.io/ceph/ceph:v18)"
+      - "check-success=raw-disk (quay.io/ceph/ceph:v18)"
+      - "check-success=two-osds-in-device (quay.io/ceph/ceph:v18)"
+      - "check-success=osd-with-metadata-partition-device (quay.io/ceph/ceph:v18)"
+      - "check-success=osd-with-metadata-device (quay.io/ceph/ceph:v18)"
+      - "check-success=encryption (quay.io/ceph/ceph:v18)"
+      - "check-success=lvm (quay.io/ceph/ceph:v18)"
+      - "check-success=pvc (quay.io/ceph/ceph:v18)"
+      - "check-success=pvc-db (quay.io/ceph/ceph:v18)"
+      - "check-success=pvc-db-wal (quay.io/ceph/ceph:v18)"
+      - "check-success=encryption-pvc (quay.io/ceph/ceph:v18)"
+      - "check-success=encryption-pvc-db (quay.io/ceph/ceph:v18)"
+      - "check-success=encryption-pvc-db-wal (quay.io/ceph/ceph:v18)"
+      - "check-success=encryption-pvc-kms-vault-token-auth (quay.io/ceph/ceph:v18)"
+      - "check-success=encryption-pvc-kms-vault-k8s-auth (quay.io/ceph/ceph:v18)"
+      - "check-success=lvm-pvc (quay.io/ceph/ceph:v18)"
+      # - "check-success=multi-cluster-mirroring (quay.io/ceph/ceph:v18)"
+      - "check-success=rgw-multisite-testing (quay.io/ceph/ceph:v18)"
+      # - "check-success=encryption-pvc-kms-ibm-kp (quay.io/ceph/ceph:v18)"
+      - "check-success=multus-cluster-network (quay.io/ceph/ceph:v18)"
+      - "check-success=csi-hostnetwork-disabled (quay.io/ceph/ceph:v18)"
```

Comment:
Added (or explicitly ignored) 5 missing canary tests to this list along with the new parenthetical "matrix" output.

Reply:
Actually how about if we just keep this change in master, and don't backport to 1.13? This means we could revert this update to the 1.13 mergify rules. But to keep this list of tests for mergify, we could create a new section for the release-1.14 branch? Mergify should just ignore it until we actually create the branch in the future.